index.js

```diff
@@ -201,6 +202,52 @@ module.exports = function(options) {
           return callback(null);
         });
       },
+      function version(callback) {
+        if (!options.version) {
+          return callback(null);
+        }
+        const packageJson = files.reduce(function(prev, file) {
+          if (file.path.indexOf('package.json') !== -1) {
+            return file.path
+          }
+        }, '');
+        var packageContent = fs.readFileSync(packageJson, 'utf8');
+        var json = JSON.parse(packageContent.toString());
+        var version = json.version;
```

Comment:
Could the version extraction be moved into it's own support function? Would make it easier to support other project types later.